src/components/People.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from "react";
+
+
+
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state = {
+            people: [],
+            userInput: ""
+        };
+    };
+
+    fetchPeople = () => {
+        fetch("https://ghibliapi.herokuapp.com/people")
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                people: data,
+
+            });
+        });
+    };
+
+    componentDidMount = () => {
+        this.fetchPeople();
+    };
+
+
+    handleOnChange = (event) => {
+        this.setState({
+            userInput: event.target.value
+        })
+
+    }
+
+    handleOnSubmit = (e) => {
+        e.preventDefault();
+        
+
+    }
+
+    render(){
+        return(
+            <div className="people">
+                <h1>
+                    Search for a Person
+                </h1>
+                <form>
+                    <input 
+                        id="user-input"
+                        type="text"
+                        placeholder="Find Your Person"
+                        onChange={this.handleOnChange}
+                    />
+                    <button onSubmit={this.handleOnSubmit}>
+                        Submit
+                    </button>
+                </form>
+
```

Comment:
```suggestion
           <div className="person-info">
                <h2>Name: {this.state.selectedPerson.name}</h2>
                <h2>Age: {this.state.selectedPerson.age}</h2>
                <h2>Gender: {this.state.selectedPerson.gender}</h2>
           </div>
```
You would need to add some conditional logic to this depending on two things:
1) if the user has entered a search yet
2) if the user's search was successful or unsuccessful

That being said, this is where you would display the `selectedCharacter`'s info. This will throw an error without conditional logic, but it's a start for now. I recommend booking office hours with me, Greg or Anthony to get more clarification on this!